compute/instances/custom-machine-type/create_with_helper.go

```diff
@@ -0,0 +1,222 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START compute_custom_machine_type_create_with_helper]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	compute "cloud.google.com/go/compute/apiv1"
+	computepb "google.golang.org/genproto/googleapis/cloud/compute/v1"
+	"google.golang.org/protobuf/proto"
+)
+
+func customMachineTypeHelper(zone, cpuSeries string, coreCount, memory int) (error, string) {
```

Comment:
Same here. Error should be the last return value.

Reply:
What's a better name for this function? Helper isn't descriptive. Perhaps `instanceURI`? Or whatever the appropriate term is for "zones/%v/machineTypes/%v-%v-%v"?